Makefile

```diff
@@ -16,3 +22,11 @@ compose-node-down:
 	@echo "[composing node down]"
 	@docker-compose -f $(CHAIN)/docker-compose.yml down
 
+build-node-runner:
+	@echo "[build node runner]"
+	@go build -o bin/$(CHAIN)/runner src/cmd/$(CHAIN)/main.go
+	@echo "Build $(CHAIN) node done ✔︎"
+
+run-node-local:
+	@echo "[run node local]"
+	@export $$(cat $(CHAIN)/node_local.env) && nodemon --exec go run src/cmd/$(CHAIN)/main.go
```

Comment:
@cagodoy suggested not using nodemon in the repo since it runs on node.

But if you will keep it, I suggest checking putting in the `Makefile` a check of wether node is or not installed, and if not install it.